js/views/contentScreen.js

```diff
@@ -45,12 +46,12 @@ var ContentScreen = React.createClass({
 
     return (
       <div className={this.props.screenClassName}>
+        <Watermark {...this.props} controlBarVisible={false} nonClickable={true}/>
```

Comment:
do we need the `nonClickable ` param? should `general.watermark.clickUrl` determine if the watermark is clickable or not?